perun-core/src/main/java/cz/metacentrum/perun/core/impl/PerunRolesLoader.java

```diff
@@ -0,0 +1,61 @@
+package cz.metacentrum.perun.core.impl;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import cz.metacentrum.perun.core.api.BeansUtils;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.core.JdbcPerunTemplate;
+import org.springframework.core.io.Resource;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+public class PerunRolesLoader {
```

Comment:
Maybe a simple class javadoc about purpose of this class and notes about where it get configuration from.

Reply:
Added